README.md

```diff
@@ -1 +1,9 @@
-# mv-trial-review-1
\ No newline at end of file
+# MV-HELLO-MICROVERSE 
+
+## Author
+
+👤 **Abimbola**
+
+- GitHub: [bimbolabuari](https://github.com/bimbolabuari)
+- Twitter: [bimbolabuari](https://twitter.com/bimbolabuari)
+- LinkedIn: [bimbolabuari](https://linkedin.com/in/bimbolabuari)
```

Comment:
Good job adding a README file to your repository 👏 . But for this project and most of the following projects during your path in Microverse you will be using [this template](https://github.com/microverseinc/readme-template). This will ensure that you're repositories remain professional.